### PR TITLE
clientBuilder, make properties in builders private static final

### PR DESCRIPTION
**Pull request by haolingdong-msft**
Fix https://github.com/Azure/autorest.java/issues/1764

### CODE REVIEW

**Review by weidongxu-microsoft — APPROVED** [review]
when done these changes on builder, please regen e.g. devcenter and see if checkstyle etc. will complain on the changes.

**Comment by haolingdong-msft** [comment]
> when done these changes on builder, please regen e.g. devcenter and see if checkstyle etc. will complain on the changes.

sure, do we just generate and create pr, but not merge it?

**Comment by weidongxu-microsoft** [comment]
> sure, do we just generate and create pr, but not merge it?

Just a draft, to verify CI.

File: Generate.ps1 [hunk 1]
```diff
@@ -11,6 +11,9 @@ $generateScript = {
     $generateOutput = Invoke-Expression "autorest $_"
     $global:ExitCode = $global:ExitCode -bor $LASTEXITCODE
 
+    if ($global:ExitCode -ne 0) {
```
Comment (haolingdong-msft) [inline Generate.ps1:14]: add this check to make the script exit when generation fails.

**Pull request merged**